### ISSUE TITLE
Incompatibility with Werkzeug >=1.0.0

### ISSUE THREAD

**Issue by ericdill** [AlbertDeFusco/tranquilizer#22]
Looks like Werkzeug changed its API in 1.0.0 and flask-restplus hasn't accounted for that. With Werkzeug 1.0.0 and 1.0.1, I get the following stack trace

```
$ tranquilizer -h
Traceback (most recent call last):                                                                                        
  File "/home/ericdill/miniconda/envs/tranq-pip/bin/tranquilizer", line 11, in <module>                                                                
    load_entry_point('tranquilizer', 'console_scripts', 'tranquilizer')()                                                                              
  File "/home/ericdill/miniconda/envs/tranq-pip/lib/python3.7/site-packages/pkg_resources/__init__.py", line 490, in load_entry_point                                
    return get_distribution(dist).load_entry_point(group, name)                                                                                        
  File "/home/ericdill/miniconda/envs/tranq-pip/lib/python3.7/site-packages/pkg_resources/__init__.py", line 2862, in load_entry_point                               
    return ep.load()                                                                                                                                   
  File "/home/ericdill/miniconda/envs/tranq-pip/lib/python3.7/site-packages/pkg_resources/__init__.py", line 2462, in load                                           
    return self.resolve()                                                                                                                              
  File "/home/ericdill/miniconda/envs/tranq-pip/lib/python3.7/site-packages/pkg_resources/__init__.py", line 2468, in resolve                                        
    module = __import__(self.module_name, fromlist=['__name__'], level=0)                
  File "/home/ericdill/dev/tranquilizer/tranquilizer/main.py", line 5, in <module>                                                                                   
    from .application import make_app                                                                                     
  File "/home/ericdill/dev/tranquilizer/tranquilizer/application.py", line 3, in <module>                                                                            
    from flask_restplus import Api, Namespace                                                                                                          
  File "/home/ericdill/miniconda/envs/tranq-pip/lib/python3.7/site-packages/flask_restplus/__init__.py", line 4, in <module>                                         
    from . import fields, reqparse, apidoc, inputs, cors                                                                                               
  File "/home/ericdill/miniconda/envs/tranq-pip/lib/python3.7/site-packages/flask_restplus/fields.py", line 17, in <module>                                          
    from werkzeug import cached_property                                                                                                               
ImportError: cannot import name 'cached_property' from 'werkzeug' (/home/ericdill/miniconda/envs/tranq-pip/lib/python3.7/site-packages/werkzeug/__init__.py)
```

Oh dang -- Looking at flask-restplus it's now unmaintained and has been forked to [flask-restx](https://github.com/python-restx/flask-restx). Were you aware of this?

I suspect two fixes are in order:

1. pin Werkzeug to 0.16.1 in the setup.py and on conda-forge.
2. Update tranquilizer to use flask-restx

I can definitely handle pinning the dep on conda-forge and may be able to work on the second item over time.

**Comment by AlbertDeFusco** [comment]
Any help you're willing to provide would be greatly appreciated. Thanks! 

Yes, we should get moved over to flask-restx.

**Comment by AlbertDeFusco** [comment]
0.4.2 has been published to pypi and conda-forge.

**Comment by AlbertDeFusco** [comment]
the switch to flask-restx was implemented in #25 

I'll get version 0.5 published soon
